engine/src/witness/btc/btc_deposits.rs

```diff
@@ -28,6 +28,7 @@ impl<Inner: ChunkedByVault> ChunkedByVaultBuilder<Inner> {
 	pub fn btc_deposits<ProcessCall, ProcessingFut>(
 		self,
 		process_call: ProcessCall,
+		is_prewitness: bool,
```

Comment:
I would say `is_prewitness` works better because to me `should_prewitness = true` implies that this would both pre-witness and post-witness, which is not the case.

Reply:
hmm maybe, I more thought `should` because it implies an action rather than a state. will leave it up to Jamie, idm either way

Reply:
I would lean towards Maxim's thinking here. I prefer `is_prewitness`